backend/coreapp/compiler_wrapper.py

```diff
@@ -179,7 +179,7 @@ def compile_code(
                 if e.stdout:
                     msg = f"{e.stdout}\n{e.stderr}"
                 else:
-                    msg = e.stderr
+                    msg = f"{e.stderr}"
```

Comment:
This format string isn't needed if we're just setting msg to one variable. But also I think we can change this whole section to just `msg = e.stdout` since stderr now no longer exists.